packages/babel-core/src/config/plugin.js

```diff
@@ -20,8 +20,11 @@ export default class Plugin {
     if (plugin.capabilities != null && !Array.isArray(plugin.capabilities)) {
       throw new Error("Plugin .capabilities must be an array");
     }
-    if (plugin.dependencies != null && !Array.isArray(plugin.dependencies)) {
-      throw new Error("Plugin .dependencies must be an array");
+    if (!Array.isArray(plugin.after)) {
+      throw new Error("Plugin .after must be an array");
+    }
+    if (!Array.isArray(plugin.before)) {
+      throw new Error("Plugin .before must be an array");
```

Comment:
Naive thinking perhaps, but judging by the conditionals above this one, it may freak out if `plugin.before` / `plugin.after` isn't present.